stations.csv

```diff
@@ -21634,7 +21634,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 23190;Virton;virton;8866407;;49.560821;5.519125;;f;BE;f;Europe/Brussels;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
 23191;Athus;athus;8866605;;49.563346;5.828947;;f;BE;f;Europe/Brussels;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;Athem;;;;;;;;;;;;;;;;
 23192;Messancy;messancy;8866662;;49.592777;5.818915;;f;BE;f;Europe/Brussels;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
-23193;Marchienne-au-Pont;marchienne-au-pont;8871100;;50.412171;4.394223;;f;BE;f;Europe/Brussels;f;;BEMAR;f;;f;8800122;t;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
+23193;Marchienne-au-Pont;marchienne-au-pont;8871100;;50.412171;4.394223;;f;BE;f;Europe/Brussels;f;BEMAR;BEMAR;f;;f;8800122;t;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;
```

Comment:
Could you remove the content of the `sncf_tvs_id` column? I'm quite sure it's wrong.